src/lib/lib.h

```diff
@@ -27,7 +27,8 @@
 #include "zend_API.h"
 #include "compat.h"
 
-#define MICRO_IN_SEC 1000000.00
+#define XDEBUG_MICRO_IN_SEC 1000000 // 1e6, must be integer
+#define XDEBUG_NANO_IN_SEC 1000000000 // 1e9, must be integer
```

Comment:
> If it's a big number, the constant should have an L at the end.

it is still below unsigned/signed int, I think we do not need `L`, do we?

> If these constants are only used in the "library"

`XDEBUG_MICRO_IN_SEC` is no longer needed because of refactoring, but `XDEBUG_NANO_IN_SEC` should stay global because it is related with our new nanotime resolution. Should prefix be dropped even ten?